src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1030,53 +1034,71 @@ static bool isOrContainsErrorType(TypeSymbol typeSymbol)
                 MethodSymbol overriddenMethod,
                 MethodSymbol overridingMethod,
                 DiagnosticBag diagnostics,
+                bool checkReturnType,
                 bool checkParameters)
             {
-                var compilation = overridingMethod.DeclaringCompilation;
-                if (overriddenMethod is null ||
-                    overridingMethod is null ||
-                    compilation is null ||
-                    !compilation.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes))
-                {
-                    // Don't do any validation if the nullable feature is not enabled or
-                    // the override is not written directly in source
-                    return;
-                }
+                CheckValidNullableMethodOverride(overridingMethod.DeclaringCompilation, overriddenMethod, overridingMethod, diagnostics,
+                                                 checkReturnType ?
+                                                    (diagnostics, overriddenMethod, overridingMethod, location) => diagnostics.Add(ErrorCode.WRN_NullabilityMismatchInReturnTypeOnOverride, location) :
```

Comment:
>overriddenMethod, overridingMethod [](start = 66, length = 34)

Consider using `_` for ignored parameter.

Reply:
> Consider using _ for ignored parameter.

I see no point in doing this, it is not hard for me to type the names and it is better for readability in my opinion. Also, I do not think both parameters can be named ```_```. 

---
In reply to: [296388073](https://github.com/dotnet/roslyn/pull/36663#discussion_r296388073) [](ancestors = 296388073)